wire-ios-automation/ios/src/main/java/com/wearezeta/auto/ios/common/IOSDriverBuilder.java

```diff
@@ -0,0 +1,224 @@
+    }
+
+    public IOSDriverBuilder withProcessArgs(String... args) {
+        for (String arg : args) {
+            processArgs.add(arg);
+        }
+        return this;
+    }
+
+    public IOSDriverBuilder withHub(URL url) {
+        this.hubUrl = url;
+        return this;
+    }
+
+    public IOSDriverBuilder withFastLoginUser(ClientUser fastLoginUser) {
+        if (fastLoginUser != null) {
+            processArgs.addAll(Arrays.asList(
+                    // https://wearezeta.atlassian.net/browse/ZIOS-6747
+                    "--loginemail=" + fastLoginUser.getEmail(),
+                    "--loginpassword=" + fastLoginUser.getPassword()
+            ));
+        }
+        return this;
+    }
+
+    public IOSDriverBuilder withLogListener(Consumer<String> logListenerHandler) {
+        this.logListenerHandler = logListenerHandler;
+        return this;
+    }
+
+    public IOSDriverBuilder withFullReset(boolean fullReset) {
+        this.fullReset = fullReset;
+        return this;
+    }
+
+    public void withMicrophoneAccess() {
+        allowMicrophoneAccess = true;
+    }
+
+    public void withCameraAccess() {
+        allowCameraAccess = true;
+    }
+
+    public void withAccessToAllPhotos() {
+        allowAccessToAllPhotos = true;
+    }
+
+    public void withNotifications() {
+        allowNotifications = true;
+    }
+
+    public void withUninstallingAllVersionsOfWire() {
+        uninstallAllOtherVersions = true;
+    }
+
+    public WebDriver build() {
+        if (hubUrl == null) {
+            throw new RuntimeException("No hub url specified.");
+        }
+
+        if (capabilities == null) {
+            throw new RuntimeException("No capabilities specified.");
+        }
+
+        if (!processArgs.contains("-BackendEnvironmentTypeOverrideKey")) {
+            String backendType = Config.current().getBackendType(this.getClass());
+            processArgs.add("-BackendEnvironmentTypeOverrideKey");
+            if (backendType.equals("qa-column-1")) {
+                // Using staging here because qa-column-1 is configured in the backend bundle for staging
+                // See: https://github.com/wireapp/wire-ios-build-assets/blob/master/BK-CI-configuration/AppStore/Backend.bundle/staging.json
+                processArgs.add("staging");
+            } else if (backendType.equals("qa-column-3")) {
+                // See: https://github.com/wireapp/wire-ios-build-assets/blob/master/COLUMN3-CI-configuration/RC/Backend.bundle/staging.json
+                processArgs.add("staging");
+            } else {
+                processArgs.add(backendType);
+            }
+        }
+
+        // TODO: Find a better way for adding this capability
+        final JSONObject processArguments = new JSONObject();
+        processArguments.put("args", processArgs);
+        // Enable logging for AVS and calling modules by default
+        processArguments.put("env", ImmutableMap.of("ZMLOG_TAGS", "AVS,calling"));
```

Comment:
we should remove this also on our side, ZMLOG is going away, the AVS one has been ported to WireLogger.
"calling" still exists but should soon be ported too